code/src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import moment from "moment";
+
+import styled from "styled-components";
+
+export const Header = () => {
+  const currentDate = moment().format("dddd, MMM Do");
+  const currentTime = moment().format("LT");
+
+  return (
+    <HeaderStyled>
+      <div>
+        <h2>MY TASKS</h2>
+        <DatenTime>
+          <p>
+            {currentDate} - {currentTime}
+          </p>
+        </DatenTime>
+      </div>
+    </HeaderStyled>
+  );
+};
+export const HeaderStyled = styled.div`
```

Comment:
I think it is best practice to declare variables before you use them, which is why it is common to have your styled components before your component 😊